Crash on startup

Couldn't figure out how to access the plugin, but upon looking at the logs it appears to be crashing during the loading process

```
2022-01-26 08:37:45 [pool-2-thread-1] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "tasks-tracker" version "48b43887" commit "48b4388744a12ef1272573d218826564f9b4e8d0"
2022-01-26 08:37:45 [pool-2-thread-1] WARN  n.r.c.e.ExternalPluginManager - Unable to start or load external plugin "tasks-tracker"
java.lang.reflect.InvocationTargetException: null
	at java.desktop/java.awt.EventQueue.invokeAndWait(Unknown Source)
	at java.desktop/java.awt.EventQueue.invokeAndWait(Unknown Source)
	at java.desktop/javax.swing.SwingUtilities.invokeAndWait(Unknown Source)
	at net.runelite.client.externalplugins.ExternalPluginManager.refreshPlugins(ExternalPluginManager.java:324)
	at net.runelite.client.util.RunnableExceptionLogger.run(RunnableExceptionLogger.java:41)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: net.runelite.client.plugins.PluginInstantiationException: java.lang.NullPointerException
	at net.runelite.client.externalplugins.ExternalPluginManager.lambda$refreshPlugins$3(ExternalPluginManager.java:335)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
Caused by: net.runelite.client.plugins.PluginInstantiationException: java.lang.NullPointerException
	at net.runelite.client.plugins.PluginManager.startPlugin(PluginManager.java:424)
	at net.runelite.client.externalplugins.ExternalPluginManager.lambda$refreshPlugins$3(ExternalPluginManager.java:330)
	... 13 common frames omitted
Caused by: java.lang.NullPointerException: null
	at net.reldo.taskstracker.tasktypes.AbstractTaskManager.applyTrackerSave(AbstractTaskManager.java:49)
	at net.reldo.taskstracker.TasksTrackerPlugin.loadProfile(TasksTrackerPlugin.java:149)
	at net.reldo.taskstracker.TasksTrackerPlugin.startUp(TasksTrackerPlugin.java:123)
	at net.runelite.client.plugins.PluginManager.startPlugin(PluginManager.java:402)
	... 14 common frames omitted
```

Plugin looks great :)

--- a/src/main/java/net/reldo/taskstracker/TasksTrackerPlugin.java
+++ b/src/main/java/net/reldo/taskstracker/TasksTrackerPlugin.java
@@ -156,7 +156,7 @@ public class TasksTrackerPlugin extends Plugin
 	protected void shutDown() throws Exception
 	{
 		pluginPanel = null;
-		taskManagers = null;
+		taskManagers = new HashMap<>();
 		clientToolbar.removeNavigation(navButton);
 		log.info("Tasks Tracker stopped!");
 	}

--- a/src/main/java/net/reldo/taskstracker/tasktypes/AbstractTaskManager.java
+++ b/src/main/java/net/reldo/taskstracker/tasktypes/AbstractTaskManager.java
@@ -20,7 +20,7 @@ public abstract class AbstractTaskManager
 	private TaskDataClient taskDataClient;
 	private final TasksTrackerPlugin plugin;
 	public TaskType taskType;
-	public ArrayList<Task> tasks;
+	public ArrayList<Task> tasks = new ArrayList<>();
 	public int maxTaskCount;
 
 	public AbstractTaskManager(TaskType taskType, TasksTrackerPlugin plugin, TrackerDataStore trackerDataStore, TaskDataClient taskDataClient)

